test/txmeta-test.js

```diff
@@ -17,9 +17,11 @@ describe('TXMeta', function() {
     assert.strictEqual(txJSON1.confirmations, 0);
 
     // confirmed once at height 100
-    const txmeta2 = TXMeta.fromOptions( {height: 100} );
+    const block = Buffer.from('058f5cf9187d9f60729245956688022474ffc0eda80df6340a2053d5c4d149af');
+    const txmeta2 = TXMeta.fromOptions( {height: 100, block} );
     txmeta2.height = 100;
     const txJSON2 = txmeta2.getJSON(network, null, 100);
     assert.strictEqual(txJSON2.confirmations, 1);
+    assert.strictEqual(txJSON2.block, block.toString('hex'));
```

Comment:
Should be in its own test. That way, if the test fails we'll know what caused the regression. You could rename the existing test `should return correct confirmations` (thats my bad :-P) and yours would be like `should get block hash as string`

Reply:
Okay perfect will do